static/app/views/alerts/rules/metric/details/body.tsx

```diff
@@ -236,13 +237,22 @@ export default function MetricDetailsBody({
             )}
           </StyledSubHeader>
 
+          {selectedIncident?.alertRule.detectionType ===
+            AlertRuleComparisonType.DYNAMIC && (
+            <AnomalyDetectionFeedbackBanner
+              key={selectedIncident.id}
+              id={selectedIncident.id}
+              organization={organization}
+              selectedIncident={selectedIncident}
+            />
+          )}
+
           <ErrorMigrationWarning project={project} rule={rule} />
 
           {/* TODO: add activation start/stop into chart */}
           <MetricChart
             api={api}
             rule={rule}
-            incidents={incidents}
```

Comment:
Intentional deletion?

Reply:
Definitely not and I was confused about why the incidents weren't rendering! Thanks for catching this 😅